tests/test_on_data/test_imaging_interfaces.py

```diff
@@ -54,3 +57,25 @@ class TestSbxImagingInterface(ImagingExtractorInterfaceTestMixin, TestCase):
         dict(file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "Scanbox" / f"sample.sbx")),
     ]
     save_directory = OUTPUT_PATH
+
+
+class TestBrukerTiffImagingInterface(ImagingExtractorInterfaceTestMixin, TestCase):
+    data_interface_cls = BrukerTiffImagingInterface
+    interface_kwargs = dict(
+        folder_path=str(
+            OPHYS_DATA_PATH / "imaging_datasets" / "BrukerTif" / "NCCR32_2023_02_20_Into_the_void_t_series_baseline-000"
+        )
+    )
+    save_directory = OUTPUT_PATH
+
+    def check_extracted_metadata(self, metadata: dict):
+        self.assertEqual(metadata["NWBFile"]["session_start_time"], datetime(2023, 2, 20, 15, 58, 25))
+        self.assertEqual(metadata["Ophys"]["Device"][0]["name"], "Bruker Fluorescence Microscope")
+        self.assertEqual(metadata["Ophys"]["Device"][0]["description"], "Version 5.6.64.400")
+        self.assertEqual(metadata["Ophys"]["ImagingPlane"][0]["imaging_rate"], 30.345939461428763)
+        self.assertEqual(metadata["Ophys"]["ImagingPlane"][0]["description"], "The plane imaged at 5e-06 meters depth.")
+        assert_array_equal(metadata["Ophys"]["ImagingPlane"][0]["grid_spacing"], [1.1078125e-06, 1.1078125e-06])
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["unit"], "px")
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["field_of_view"], [0.0005672, 0.0005672, 5e-06])
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["format"], "tiff")
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["scan_line_rate"], 15840.580398865815)
```

Comment:
For field of view and scan line rate, can you check they actually end up in the NWB file?

And check all this metadata stuff in the actual NWB file as well as in the returned metadata structure?